Documentation/ceph-nfs-crd.md

```diff
@@ -72,55 +83,398 @@ spec:
     priorityClassName:
 ```
 
- Enable the creation of NFS exports in the dashboard for a given cephfs or object gateway pool by running the following command in the toolbox container:
+For Ceph v16 and above, the `rados` spec can be omitted entirely, and no CephFilesystem or
+CephObjectStore are necessary for storing NFS configuration data.
+
+## NFS Settings
+
+### RADOS Settings
+NFS configuration is stored in a Ceph pool so that it is highly available and protected. How that is
+configured changes depending on the Ceph version. Configuring the pool is done via the `rados` config.
+
+> **WARNING**: Do not use error corrected (EC) pools for NFS. NFS-Ganesha uses OMAP which is not
+> supported by Ceph's error correction.
+
+#### For Ceph v15
+* `pool`: (mandatory) The Ceph pool where NFS configuration is stored.
+* `namespace`: (mandatory) The namespace in the `pool` where configuration objects will be stored.
+
+#### For Ceph v16 or newer
+* `poolConfig`: (optional) The pool settings to use for the RADOS pool.
+  It matches the [CephBlockPool](ceph-block.md) specification.
+  The settings will be applied to a pool named `.nfs` on Ceph v16.2.7 or newer.
+
+Rook ignores both `pool` and `namespace` ([see above](#for-ceph-v15)) settings when running Ceph v16
+or newer.
+
+
+## Creating Exports
+When a CephNFS is first created, all NFS daemons within the CephNFS cluster will share a
+configuration with no exports defined.
+
+### For Ceph v15
```

Comment:
Do we need to document v15 exports? Seems like we should concentrate docs on 16.2.7+.

Reply:
We still support v15, and I still took the time to write the documentation, so I don't see the hurt.

Reply:
This looks more like a Ceph documentation rather than a Rook at this point 😅